go/multipooler/manager/manager.go

```diff
@@ -970,7 +911,15 @@ func (pm *MultiPoolerManager) setSynchronousCommit(ctx context.Context, synchron
 // Note: Use '*' to match all connected standbys, or specify explicit standby application_name values
 // Application names are generated from multipooler IDs using the shared generateApplicationName helper
 func (pm *MultiPoolerManager) setSynchronousStandbyNames(ctx context.Context, synchronousMethod multipoolermanagerdata.SynchronousMethod, numSync int32, standbyIDs []*clustermetadatapb.ID) error {
```

Comment:
Higher level style question: do we want do something like a "Locked" suffix for method that should be called when a lock is already held, and if so would this method be appropriate for that sort of suffix?